R/module_data_summary.R

```diff
@@ -65,24 +71,17 @@ srv_data_summary <- function(id, teal_data) {
         if (!length(ls(teal.code::get_env(teal_data())))) {
           return(NULL)
         }
+        get_filter_overview_wrapper(teal_data)
+      })
 
-        filter_overview <- get_filter_overview(teal_data)
-        names(filter_overview)[[1]] <- "Data Name"
-
-        filter_overview$Obs <- ifelse(
-          !is.na(filter_overview$obs),
-          sprintf("%s/%s", filter_overview$obs_filtered, filter_overview$obs),
-          ifelse(!is.na(filter_overview$obs_filtered), sprintf("%s", filter_overview$obs_filtered), "")
-        )
-
-        filter_overview$Subjects <- ifelse(
-          !is.na(filter_overview$subjects),
-          sprintf("%s/%s", filter_overview$subjects_filtered, filter_overview$subjects),
-          ""
-        )
-
-        filter_overview <- filter_overview[, colnames(filter_overview) %in% c("Data Name", "Obs", "Subjects")]
-        Filter(function(col) !all(col == ""), filter_overview)
+      is_unsupported_hidden <- reactive({
+        is_hidden <- isTRUE(input$toggle_unsupported %% 2 == 0)
```

Comment:
this is not very clear
did you consider toggle type of input instead of action link?
then the whole update of labels would not be needed

Reply:
What do you mean by type of input?

Reply:
I mean `shiny::actionLink` vs `shiny::checkboxInput` (or `shiny::input_switch`)

Reply:
Thanks for pushing this. It makes it a little bit simpler now